openapi/openapi/src/main/scala/endpoints4s/openapi/JsonSchemas.scala

```diff
@@ -83,7 +88,12 @@ trait JsonSchemas extends algebra.JsonSchemas with TuplesSchemas {
         format: Option[String] = None,
         description: Option[String] = None,
         example: Option[ujson.Value] = None,
-        title: Option[String] = None
+        title: Option[String] = None,
+        minimum: Option[Double] = None,
+        exclusiveMinimum: Option[Boolean] = None,
+        maximum: Option[Double] = None,
+        exclusiveMaximum: Option[Boolean] = None,
+        multipleOf: Option[Double] = None
```

Comment:
This change breaks backward binary compatibility. We need to bump the major version of all the interpreters.

Reply:
All? The others don't depend on the openapi right? Do we need to also make normal classes for these (maybe in a separate PR)?

Reply:
All the server interpreters depend on the `openapi` interpreter, so we’ll have to bump them as well.

Yes, turning these classes into normal classes would help, but IMHO major version bumps in the interpreters is not an issue, what’s really important is to keep backward binary compatibility for the algebra as long as possible (because the algebra is the artifact that is shared between multiple services, in a microservices architecture, and we don’t want to create incompatibilities between microservices).

Reply:
Ah right that makes sense. I don't know where to update the version though :sweat_smile: Btw the Numeric problem is solved, though there is some other error that I occasionally see (something with a server not responding).

Reply:
I’ve bumped the server interpreters, but I’ve just realized that client interpreters also depend on the `openapi` module, for some reason, so we’ll have to bump them as well…

Reply:
Shouldn't we check whether that is necessary? It seems to be that both server and clients are independent, though I might be missing something obvious.

Reply:
Well, the clients don’t really use the OpenAPI stuff, but the `JsonSchemas` interpreter based on `ujson` to decode and encode JSON entities. I think it’s OK to bump their version as well.